test/unit/html_test.rb

```diff
@@ -0,0 +1,331 @@
+require 'test_helper'
+
+describe RichText::HTML do
+
+  it 'renders basic paragraph string' do
+    d = RichText::Delta.new([{ insert: "hello world\n" }])
+    assert_equal '<p>hello world</p>', render_compact_html(d)
```

Comment:
awesome tests! really cool tool. where in this project are deletions handled?

`{ insert: 'a' }` = `<p>a</p>` but i delete `a` and insert `b` so the it overwrites `<p>a</p>` with `<p>b</p>`?

Reply:
Similar to https://github.com/voxmedia/convert-rich-text, this is an insert-only renderer. You'll see it [errors](https://github.com/voxmedia/rich-text-ruby/pull/6/files#diff-6c620e9361f3933df560d1ea01e45db7R31) if you try to run it on a transitional delta.